src/main/kotlin/org/wfanet/measurement/duchy/daemon/mill/liquidlegionsv2/LiquidLegionsV2Mill.kt

```diff
@@ -636,6 +640,12 @@ class LiquidLegionsV2Mill(
           .token
       }
 
+    // If this is a reach-only computation, then our job is done.
+    if (getMaximumRequestedFrequency(token) == 1) {
+      sendResultToKingdom(token, reach, emptyMap<Long, Double>())
```

Comment:
Would it make more sense to output a map from 1 to 1 (i.e. 100% has frequency >= 1) here?